common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -1520,7 +1520,7 @@ def reset_problem(self, _data):
             return {
                 'success': False,
                 # Translators: 'closed' means the problem's due date has passed. You may no longer attempt to solve the problem.
-                'error': _("Problem is closed."),
+                'msg': _("Problem is closed."),
```

Comment:
Since I'm not sure we want to be as specific as saying that "closed" always means the due date has passed, I'd just add "This": -> "This problem is closed."